pkg/helm_actions/list_releases.go

```diff
@@ -0,0 +1,17 @@
+package helm_actions
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+)
+
+func ListReleases(conf *action.Configuration) (interface{}, error) {
+	cmd := action.NewList(conf)
+	cmd.AllNamespaces = true
```

Comment:
releases have to be listed in the UI in the namespace context primarily.  How about doing a 
```
cmd := action.ListReleases(conf,ns)
```

Reply:
namespace will now be set upfront in `conf` itself